NAV_Algorithms/flight_observer.h

```diff
@@ -85,21 +85,26 @@ class flight_observer_t
 
 	float get_pressure_altitude( void) const;
 
-	float get_speed_compensation( unsigned index) const
-	{
-	  switch( index)
-	  {
-	    case 0:
-		return speed_compensation_INS_GNSS_1;
-	      break;
-	    case 1:
-		return speed_compensation_kalman_2;
-	      break;
-	    default:
-		return speed_compensation_energy_3;
-	      break;
-	  }
-	}
+  float
+  get_speed_compensation (unsigned index) const
+  {
+    switch (index)
```

Comment:
Is there a description for the different speed compensation algorithms?    When to chose which one?

Reply:
This is only for development using the SIL.
In the sensor firmware all of them are used simultaneously.